pkg/volume/volume.go

```diff
@@ -92,6 +92,15 @@ type Metrics struct {
 	// a filesystem with the host (e.g. emptydir, hostpath), this is the free inodes
 	// on the underlying storage, and is shared with host processes and other volumes
 	InodesFree *resource.Quantity
+
+	// Normal volumes are available for use and operating optimally.
+	// An abnormal volume does not meet these criteria.
+	// This field is REQUIRED.
```

Comment:
I guess it's required in the csi spec if you support the capability, but from a k8s volume pov, it's an optional field that's only filled in by some csi drivers right?